Server/ReasnAPI/ReasnAPI/Services/TagService.cs

```diff
@@ -22,7 +22,7 @@ public TagDto CreateTag(TagDto tagDto)
         return tagDto;
     }
 
-    public TagDto UpdateTag(int tagId, TagDto tagDto, int eventId)
+    public TagDto UpdateTagForEvent(int tagId, TagDto tagDto, int eventId)
```

Comment:
Why is it needed? We don't update tags. We remove old add new only. Only way to update tags might be for admins to change the content of the tag

Reply:
Like it is in UpdateTag, good

Reply:
Is it even used?

Reply:
its not used i removed it